api/distributedContext/context.go

```diff
@@ -12,21 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tag
+package distributedContext
```

Comment:
I wanted to discuss the guidance here:
https://blog.golang.org/package-names

specifically, 
> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

I have wanted to use a shorter name like "dctx".

Reply:
Oh yes. I'd be in full support of that. Anywhere it was used I wanted to rename it inline to dctx. distributedContext is just so long.

Reply:
Before I do, I'm curious if other people agree.

Reply:
From the spec:
// This package layout is intentionally generic and it doesn't try to impose a language specific package structure.

So IMO I think we should do the right thing for the language.

Reply:
Please don't name this "distributedContext". "distributedcontext" or "dctx" is ++ IMO.

Reply:
FWIW: I would prefer "distributedcontext"

Reply:
@freeformz , do you prefer `dctx` or `distributedcontext` ?